summer2022.iml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If this file is empty, I would recommend deleting it and/or adding to .gitignore